### DISCUSSION TITLE
React 19 / Nextjs 15 Compatibility

### DISCUSSION THREAD

**Discussion by gsw85** [wevm/wagmi#4474] [General]
Does Wagmi compatible with React 19 / Nextjs 15? I got the TypeError: storage.removeItem is not a function , Object.removeItem
, Object.onAccountsChanged on the Dev Mode

**Comment by Emanuele-Fortunati** [comment]
@gsw85 it does, but you have to use the CookieStorage. Check this out: https://wagmi.sh/react/guides/ssr#_1-set-up-cookie-storage

**Reply by gsw85** [reply]
Cool! fix the problem !
